src/main/java/jackyy/dimensionaledibles/block/BlockCustomCake.java

```diff
@@ -149,11 +191,27 @@ public TileEntity createNewTileEntity(@Nonnull World world,
 
     private final ModConfig.CakeConfig conf = new ModConfig.CakeConfig() {
         @Override
-        public String fuel() { return cakeFuel; }
+        public String fuel(int dim) {
+            if(cache.containsKey(dim))
+                return cache.get(dim).cakeFuel;
+            else
+                return "minecraft:air";
```

Comment:
This is causing the tooltip to display as "Refill using Air" instead of the bad fuel localized message. This is because `minecraft:air` is not `null` or `""`, so getFuelItemStack does not fall back to its default item, and so the custom cake does not display the error message. 
Options: make this `""` or `null`, or add `minecraft:air` to the list of fuels that will cause cakes to fall back to the default fuel.

Reply:
Should be resolved as of 7bc67e9